app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb

```diff
@@ -30,6 +31,7 @@ def self.params_for_create
           :label        => _('Cloud Database Instance Class'),
           :includeEmpty => true,
           :isRequired   => true,
+          :validate   => [{:type => 'required'}],
```

Comment:
Actually just noticed the rubocop warnings still aren't addressed, can you fix those?
```suggestion
          :validate     => [{:type => 'required'}],
```